Cargo.toml

```diff
@@ -5,7 +5,6 @@ members = [
 	"contracts/pre-propose/*",
 	"contracts/staking/*",
 	"contracts/voting/*",
-	"contracts/external/*",
```

Comment:
probably a relic from a rebase, but think we want to build these :) this is the admin factor and token swap contracts